### PR TITLE
Scala 2.13.4

### PR DESCRIPTION
**Pull request by ddworak**

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -16,7 +16,7 @@ object Dependencies {
   val scalaCssVersion = "0.7.0"
 
   val servletVersion = "4.0.1"
-  val avsCommonsVersion = "2.0.0-M16"
+  val avsCommonsVersion = "2.0.0-M18"
```
Comment (sebaciv) [inline project/Dependencies.scala:19]: 2.0.0-M19

Comment (mereszeta) [reply project/Dependencies.scala:19]: could even be M21

File: core/.js/src/main/scala/io/udash/view/ViewRenderer.scala [hunk 2]
```diff
@@ -61,7 +62,7 @@ private[udash] class ViewRenderer(rootElement: => Element) {
       case c: ContainerView => c.renderChild(None)
       case _ =>
     }
-    views.trimEnd(views.length - unmodifiedViews)
+    views.trimEnd(views.length - unmodifiedViews): @nowarn
```
Comment (mereszeta) [inline core/.js/src/main/scala/io/udash/view/ViewRenderer.scala:65]: why not `dropRightInPlace` instead?

**Review by mereszeta — APPROVED** [review]
LGTM with minor

File: project/Dependencies.scala (hunk 1 above)
Comment (ddworak) [reply project/Dependencies.scala:19]: The slower the review, the further this gets :smile:

File: core/.js/src/main/scala/io/udash/view/ViewRenderer.scala (hunk 2 above)
Comment (ddworak) [reply core/.js/src/main/scala/io/udash/view/ViewRenderer.scala:65]: If I'm not missing anything there's no easy way to provide it for 2.12 without making things ugly for 2.13.

**Pull request merged**